Bugfixing january happy new years

Volttsie opened the pull request:
Opening up january bugfixing branch + introduced greater hungary core decision improvements

Volttsie committed:
Greater Hungary adjustments

Fixed Reaffirm territorial claims decisions to reflect actual territorial claims and added decision for Prekmurje

Volttsie committed:
More Greater Hungary fixes

Added cores to Uzhorod when integrating carpathian ruthenia

Volttsie committed:
Fixes to Siberia, improvements in LAT and EST economic trees

Fixed missing states from Siberia formable, adjusted focus times for Estonia and Latvia especially for industry branch

marijn211 committed:
Update NOR.txt

DutchPlanDerLindeSus committed:
WuW-related bugs et al

DutchPlanDerLindeSus committed:
Update r56_idea_PAK_karachi_shipyard.dds

DutchPlanDerLindeSus committed:
Update HUN - Hungary.txt

DutchPlanDerLindeSus committed:
More fixes and other stuff

Volttsie committed:
Fixes to various formables

Fixes and adjustments for: Turan, Rome, Macedonia

Volttsie committed:
One more formable fix cause am stupid

Fixed Turan missing one state in Kazakhstan

BiccyThiccy committed:
Hiiiii Dutch :)

marijn211 committed:
Update NOR.txt

marijn211 committed:
Compliance for Rwanda/Burundi through hidden event

marijn211 reviewed (dismissed):
.vs folder somehow included

marijn211 committed:
Remove .vs folder

Volttsie committed:
Fixes to various nations cores and claims and events

Fixed indonesia not being able to core west guinea through pan-malay formable, fixed Bulgarian claims to both match up with irl and vanilla, fixed Austria losing it's core on South Tyrol, fixed Brest-litovsk event for germany

Volttsie committed:
Merge branch 'bugfixing-january-HappyNewYears' of https://github.com/deliciousmods/1956_beta into bugfixing-january-HappyNewYears

marijn211 committed:
Fixed Denmark wrong states

marijn211 committed:
Fix Iran becoming "Islamic Socialism"

marijn211 committed:
Stop Albania going for a wargoal against ITA on hist

marijn211 committed:
Update GER_historical_strategy_plan.txt

marijn211 committed:
tiny error fix

Volttsie committed:
Bulgarian claim changes

further updates to Bulgarian claim changes.

marijn211 committed:
Fixed some Belgian tree errors

marijn211 committed:
Merge branch 'Beta-Edits' into bugfixing-january-HappyNewYears
